### PR TITLE
[CELEBORN-1083] Refine TimeSlidingHub code

### PR DESCRIPTION
**Pull request by onebox-li**
### What changes were proposed in this pull request?
Refine some TimeSlidingHub code.

### Why are the changes needed?
Ditto

### Does this PR introduce _any_ user-facing change?
No

### How was this patch tested?
UT.

### CODE REVIEW

**Comment by onebox-li** [comment]
cc @AngersZhuuuu @boneanxs PTAL if have time :)

File: worker/src/main/java/org/apache/celeborn/service/deploy/worker/congestcontrol/TimeSlidingHub.java [hunk 1]
```diff
@@ -104,7 +103,7 @@ public synchronized void add(long currentTimestamp, N newNode) {
         _deque.add(lastNode);
```
Comment (RexXiong) [inline worker/src/main/java/org/apache/celeborn/service/deploy/worker/congestcontrol/TimeSlidingHub.java:103]: Seems all empty nodes have same timestamp? Maybe lastNode.getLeft() + intervalPerBucketInMills * index ?

Comment (onebox-li) [reply worker/src/main/java/org/apache/celeborn/service/deploy/worker/congestcontrol/TimeSlidingHub.java:103]: It seems the empty node has updated timestamp here. You can take a look at https://github.com/apache/incubator-celeborn/blob/ced12a18cd6e5c9500a25615c5e82e05bebde49f/worker/src/main/java/org/apache/celeborn/service/deploy/worker/congestcontrol/TimeSlidingHub.java#L102

Comment (RexXiong) [reply worker/src/main/java/org/apache/celeborn/service/deploy/worker/congestcontrol/TimeSlidingHub.java:103]: OK

**Review by RexXiong — APPROVED** [review]
LGTM, thanks!

**Comment by RexXiong** [comment]
Merging to main(v0.4.0)/0.3(v0.3.2)

**Pull request closed without merging**